File: programmers/java/lev0/문자 반복 출력하기.java
class Solution {
    public String solution(String my_string, int n) {
        String answer = "";
        for(String el : my_string.split(""))
            answer +=(el.repeat(n));
        
        return answer;
    }
}

// class Solution {
//     public String solution(String my_string, int n) {
//         StringBuilder sb = new StringBuilder();
//         for(char c : my_string.toCharArray()){
//             sb.append((c + "").repeat(n));
//         }
//         return sb.toString();
//     }
// }

File: programmers/js/lev0/문자 반복 출력하기.js
function solution(my_string, n) {
  return my_string
    .split("")
    .map((el) => el.repeat(n))
    .join("");
}

// 구조분해할당!!!!!!!!!!!!!!!!
function solution(my_string, n) {
  return [...my_string].map((el) => el.repeat(n)).join("");
}
